nbgrader/nbextensions/static/formgrader/main.js

```diff
@@ -0,0 +1,21 @@
+define(function(require) {
+    var $ = require('jquery');
+    var Jupyter = require('base/js/namespace');
+
+   function load() {
+        if (!Jupyter.notebook_list) return;
+        var base_url = Jupyter.notebook_list.base_url;
+        $("#tabs").append(
+            $('<li>')
+            .append(
+                $('<a>')
+                .attr('href', '/formgrader')
```

Comment:
This path can be configured on the server extension side. Can it also be configured through the javascript? If not, then it should probably not be configurable on the server extension side at all, because if it is changed there then this will just break.